01_install_requirements.sh

```diff
@@ -62,3 +62,10 @@ fi
 curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
 curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
 sudo yum install yarn nodejs -y
+
+# Install go dep command.  The packaged version is too old.  This is for development only atm.
+export GOPATH=$HOME/go
```

Comment:
Rather than hard-coding this, should we either do eval "$(go env)" or source ocp_install_env.sh which already does that?

Reply:
+1 to `eval "$(go env)"`